cli/internal/taskhash/taskhash.go

```diff
@@ -192,8 +197,9 @@ func (th *Tracker) CalculateFileHashes(
 		hashTasks.Add(pfs)
 	}
 
-	hashes := make(map[packageFileHashKey]string)
-	hashQueue := make(chan *packageFileSpec, workerCount)
+	hashes := make(map[PackageFileHashKey]string)
+	hashObjects := make(map[PackageFileHashKey]map[turbopath.AnchoredUnixPath]string)
```

Comment:
We could add a size hint of `len(hashTasks)` to these maps: https://github.com/uber-go/guide/blob/master/style.md#specifying-map-capacity-hints

Reply:
done (also added to the line above this one)